### PR TITLE
fix: add hint for password-confirmation

### PR DESCRIPTION
**Pull request by you1996**
#### Changes Preview:

afterchanges2

afterchanges

afterchangesDeutch

### CODE REVIEW

**Comment by fgerschwiler** [comment]
@you1996 Thanks for your pr.

As you requestd if there are some other related issues, beside the registration we have some more screens in the login where we check the password.
Password Change: https://github.com/caos/zitadel/blob/main/internal/ui/login/static/templates/change_password.html
Initialize Password: https://github.com/caos/zitadel/blob/main/internal/ui/login/static/templates/init_password.html 
Initialize User: https://github.com/caos/zitadel/blob/main/internal/ui/login/static/templates/init_user.html

And also in the Zitadel Console (angular) where the user can change his pw:
https://github.com/caos/zitadel/tree/main/console/src/app/pages/users/user-detail/password

File: internal/ui/login/static/i18n/de.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ Password:
   HasLowercase: Kleinbuchstaben
   HasNumber: Nummer
   HasSymbol: Symbol
+  Confirmation: Bestätigung überein
```
Comment (fgerschwiler) [inline internal/ui/login/static/i18n/de.yaml:31]:
```suggestion
  Confirmation: Bestätigung stimmt überein
```

**Comment by fgerschwiler** [comment]
@you1996 do you implement the angular part as well? or should we merge this part? and we will implement the angular part in a separat branch?

**Comment by you1996** [comment]
Hi @fgerschwiler , i think we should merge this one and fix the console(angular) part in another branch, i discovered many bugs there.
#### The "save" button is enabled even when pwc is not equal to the new password 
confirmation_issue
#### the confirmation mismatch error is there, but never printed to the user, we should fix the code that handles it
console

#### I'll fix this code in another branch today

Thanks

**Review by fgerschwiler — APPROVED** [review]

**Comment by fforootd** [comment]
Hi @you1996 we are currently fixing the CI Process for external contribution.

We would like to send you some swag  😄 (T-Shirt, Umbrella, Sticker, ...) 
If you are interested please contact me at florian@caos.ch with your address

**Comment by you1996** [comment]
Hi @fforootd, yes absolutely , thank you 😃 !

**Comment by fgerschwiler** [comment]
Hi @you1996, we just fixed our pipeline, now we are able to run it on forked repositories. 
Can you please merge our main branch to your branch? Afterwards we will be able to merge it

**Comment by livio-a** [comment]
Hi @you1996 we just deployed some fixes including this one to zitadel.ch. Thanks a lot for contributing!

**Pull request merged**